python/ray/train/lightning/_lightning_utils.py

```diff
@@ -21,12 +21,21 @@
 LIGHTNING_REPORT_STAGE_KEY = "_report_on"
 
 
+def _get_worker_root_device():
```

Comment:
nit: Does this need to be underscore prefixed?

Reply:
Oh I was trying to indicate that this function is used internally. But since we didn't expose it to the users, and internal functions in other similar modules don't have a leading underscore, I'm also going to remove it.

Reply:
Yeah I also saw this file itself has a leading underscore, so I'm assuming everything in it is supposed to be internal?

Reply:
Yeah I think so:) I've updated the code.